lib/copy_peste/command/list_result.rb

```diff
@@ -0,0 +1,60 @@
+module CopyPeste
+  class Command
+    module ListResult
+
+      # Method used to display result list of module.
+      #
+      # @param result [Collection] collection data of result list
+      # @param name [String] Name of module, could be nil
+      def display_results(results, name)
+        results = (name && !name.empty?) ? results.where(:module_name => name.upcase) : results
+        unless results.empty?
+          @graph_com.display(10, "Module\t| Date\n------\t| ----")
+          results.each do |data|
+            @graph_com.display(10, "#{data.module_name}\t| #{data.created_at}")
+          end
+        else
+          @graph_com.display(12, "Option fail, name #{name} not found") 
+        end
+      end
+      
+      # Allows a user to diplay result list generated by analysis module
+      def run
+        options = {
+          "n" => {
+            helper: "Name of module",
+            value: ''
+          },
+        }
+
+        name = ''
+        # Option set name
+        if @opts.length == 2
```

Comment:
This if shouldn't be done. When this module will have more options, some might be optional so we can't have a check on option length. First check for nonexistent options and then set authorized option to the good value. Please update it even if we'll change that in the future. Indeed, this check should be done Framework side. I'll talk to @ferrery1.